dev/tests/integration/testsuite/Magento/SendFriend/_files/product_simple.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\Catalog\Api\Data\ProductTierPriceExtensionFactory;
+use Magento\Catalog\Api\Data\ProductExtensionInterfaceFactory;
+
+\Magento\TestFramework\Helper\Bootstrap::getInstance()->reinitialize();
```

Comment:
Why do we need to have this fixture if we already have same fixture in Magento code base?